### ISSUE TITLE
BASE64URL example's miss

### ISSUE THREAD

**Issue by sh4869** [dart-lang/sdk#26353]
https://github.com/dart-lang/sdk/blob/master/sdk/lib/convert/base64.dart#L31-L34

This example should be 

```dart
 var encoded = BASE64URL.encode([0x62, 0x6c, 0xc3, 0xa5, 0x62, 0xc3, 0xa6,
                              0x72, 0x67, 0x72, 0xc3, 0xb8, 0x64]);
 var decoded = BASE64URL.decode("YmzDpWLDpnJncsO4ZAo=");
```

**Comment by lrhn** [comment]
Bad copy from `BASE64` missing the `URL` in `BASE64URL` in the example.

**Comment by floitschG** [comment]
Thanks.
Fixed in c9397d0493ae71b7c252bca690d8de6281e3740b.
